test/integration/authenticated_user_cannot_act_as_a_admin_test.rb

```diff
@@ -0,0 +1,15 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.